index.bs

```diff
@@ -385,15 +385,15 @@ like
 
 <pre><code class="lang-javascript">
   class ReadableStream {
-    constructor(underlyingSource = {}, { size, highWaterMark } = {})
+    <a href="#rs-constructor">constructor</a>(<a href="#rs-constructor">underlyingSource</a> = {}, { size, highWaterMark } = {})
```

Comment:
Again the second link to the exact same place feels pretty redundant.

Maybe what this is pointing toward is the need for a dedicated section describing the underlying source objects, instead of putting it into the constructor's intro note? I guess same for the strategies too? I'd be happy to work on that as a follow-up. In the meantime I'm indifferent as to whether we include these redundant links or not.

Reply:
The more I work on this, the more I think that’s the actual crux: Something as integral to developing streams shouldn’t just be defined in a note. So I’m up for formalizing `underlyingSource` and `underlyingSink`, but defo need some guidance what the right formalism is here.

I’ll remove the links.